doc/api/api_changes/2018-08-17-AL-deprecations.rst

```diff
@@ -0,0 +1,11 @@
+API deprecations
+````````````````
+
+The following API elements are deprecated:
+
+- ``backend_gtk3cairo.FigureCanvasGTK3Cairo``,
+- ``backend_wx.debug_on_error``, ``backend_wx.fake_stderr``,
+  ``backend_wx.raise_msg_to_str``, ``backend_wx.MenuButtonWx``,
+  ``backend_wx.PrintoutWx``,
+- ``tk_window_focus``,
+- ``mlab.demean``,
```

Comment:
What's this to do with the PR?

Reply:
leftover of when I had a single file with all deprecations. removed.